test/voting.js

```diff
@@ -415,10 +404,11 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
         })
 
         it('fails creating a vote if token has no holder', async () => {
-            await assertRevert(voting.newVote(EMPTY_SCRIPT, 'metadata'), errors.VOTING_NO_VOTING_POWER)
+            await assertRevert(voting.newVote(EMPTY_SCRIPT, 'metadata', true), errors.VOTING_NO_VOTING_POWER)
         })
     })
 
+    // TODO: Delete the bottom 2 context's worth of tests?
```

Comment:
It may make sense to remove them as all are focused on early execution. 
But also doesn't hurt to have them. Up to you!